internal/types/regex.go

```diff
@@ -32,8 +47,15 @@ func (r Regex) Compile() (*regexp.Regexp, error) {
 		switch o {
 		case 'i':
 			opts += "i"
+		case 'm':
+			opts += "m"
+		case 's':
+			opts += "s"
```

Comment:
```suggestion
		case 'i', 'm', 's':
			opts += o
```

(not tested, but should work)

Reply:
Fixed